doc/string/length.rdoc

```diff
@@ -2,11 +2,12 @@ Returns the count of characters (not bytes) in +self+:
 
   'foo'.length        # => 3
   'тест'.length       # => 4
-  'こんにちは'.length   # => 5
+  'こんにちは'.length  # => 5
```

Comment:
This breaks the alignment rendered:

Before:

<img width="814" height="378" alt="Screenshot 2025-08-25 at 9 39 01 AM" src="https://github.com/user-attachments/assets/1ad2966c-aa5c-471e-b1be-975ea9f0fd14" />

After:

<img width="801" height="436" alt="Screenshot 2025-08-25 at 9 39 15 AM" src="https://github.com/user-attachments/assets/35a80e6a-d46f-480a-901f-60828944bf18" />